apps/gateway/package.json

```
@@ -17,10 +17,15 @@
     "start": "./dist/server"
   },
   "dependencies": {
+    "@ai-sdk/amazon-bedrock": "^3.0.45",
+    "@ai-sdk/google-vertex": "^3.0.51",
+    "@ai-sdk/groq": "^2.0.28",
```

Biome (2.1.2)

[error] The key @ai-sdk/groq was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)